packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -543,7 +543,7 @@ async function downloadOffline() {
   }
   for (var resourceKey in Object.keys(RESOURCES)) {
     if (!currentContent[resourceKey]) {
-      resources.add(resourceKey);
+      resources.push(resourceKey);
     }
   }
   return Cache.addAll(resources);
```

Comment:
@jonahwilliams 
This line also causes an error when installing the service worker.

```suggestion
  return contentCache.addAll(resources);
```

Reply:
thanks!